docker/Dockerfile

```diff
@@ -26,18 +26,12 @@ WORKDIR /app
 COPY --from=build-env /app/api.out /app/api
 RUN chmod 555 /app/api
 
-RUN apk add --no-cache bash ca-certificates wget python gettext && \
-    wget -nv "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" && \
-    unzip awscli-bundle.zip && \
-    ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws && \
-    /usr/local/bin/aws --version
```

Comment:
You're going to need some of those packages I think

Reply:
Yeah, add the line `RUN apk add --no-cache bash ca-certificates` after you deleted the above.

Reply:
:+1: